frontend/src/components/ExternalDb.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section extended-section">
+    <div v-if="externalDb" class="container is-fullhd">
+      <h3 class="title">
+        {{ externalDb.dbName }} {{ components[0].componentType }} {{ externalDb.externalId }}
+      </h3>
+      <h4 class="subtitle">
+        The following Metabolic Atlas
+        {{ components.length === 1 ? 'component is' : 'components are' }}
+        associated with this external DB reference.
```

Comment:
```suggestion
        This external database reference is associated with the following Metabolic Atlas component(s):
```

Reply:
I like this suggestion! I would prefer to do the work for the user and handle the pluralizing for them though, so we show the extra `s` conditionally.